plasmapy/classes/tests/test_Plasma.py

```diff
@@ -97,6 +97,84 @@ def test_Plasma3D_derived_vars():
     assert np.allclose(test_plasma.alfven_speed.value, 10.92548431)
 
 
+class Test_PlasmaBlobRegimes():
+    def test_intermediate_and_both_energies(self):
+        T_e = 25 * 15e3 * u.K
+        n_e = 1e26 * u.cm ** -3
+        Z = 2.0
+        particle = 'p'
+        blob = plasma.PlasmaBlob(T_e=T_e,
+                                 n_e=n_e,
+                                 Z=Z,
+                                 particle=particle)
+
+        regimes = ['Intermediate coupling regime: Gamma = 10.585076050938532.',
+                   'Both Fermi and thermal energy important: Theta = 0.04113394941084522']
+
+        assert blob.regimes() == regimes
```

Comment:
Can you add an errStr here so that a clear message shows up for when this test fails? It should be in f-string format and should include the expected output along with output from `blob.regimes()`